gapic/samplegen/samplegen.py

```diff
@@ -13,29 +13,50 @@
 # limitations under the License.
 
 import itertools
+import jinja2
 import keyword
 import re
-import gapic.samplegen.utils
+
+from gapic.samplegen import utils
 
 from collections import (defaultdict, namedtuple)
-from typing import (List, Mapping, Set)
+from typing import (List, Mapping, Set, Tuple)
 
 # Outstanding issues:
-# * Neither license nor copyright are defined
 # * In real sample configs, many variables are
 #   defined with an _implicit_ $resp variable.
 
 MIN_SCHEMA_VERSION = (1, 2, 0)
 
 VALID_CONFIG_TYPE = 'com.google.api.codegen.SampleConfigProto'
 
-# There are a couple of other names that should be reserved
-# for sample variable assignments, e.g. 'client', but there are diminishing returns:
-# the sample config author isn't a pathological adversary, they're a normal person
-# who may make mistakes occasionally.
-RESERVED_WORDS = frozenset(itertools.chain(keyword.kwlist, dir(__builtins__)))
-
-TEMPLATE_NAME = 'samplegen_template.j2'
+FILE_HEADER = {"copyrightLines": ["Copyright (C) 2019  Google LLC"],
```

Comment:
In this instance it's a pure formality, but if/when the time comes to read these from files the separation makes more sense. It also makes the change for a new year more straightforward.

Reply:
Referenced below in a comment chain with @lukesneeringer : I move that the copyright and license are removed for the time being brought back upon discussion with Yoshi on where to get the information/who is responsible for it.

Reply:
Let me TAL at those comments, but I am inclined, in order of preference, (a)to not remove them for now so that by default they're there (we can alter the mechanism later) or (b) to have a big ugly hulking TODO to minimize (but I don't dare hope eliminate) the chance of us forgetting to add the license before release.

Reply:
Went with (b) in the most recent commit.